### PR TITLE
[wallet-specs] Monorepo cleanups

### PR DESCRIPTION
**Pull request by snario**
- Adds eslint
- GItignores built files
- Adds some dependencies

### CODE REVIEW

File: packages/wallet-specs/package.json [hunk 1]
```diff
@@ -24,6 +34,17 @@
     "tslint": "5.20.0",
     "tslint-config-prettier": "1.18.0",
     "tslint-plugin-prettier": "2.0.1",
-    "typescript": "^3.7.2"
+    "typescript": "^3.7.2",
+    "@typescript-eslint/eslint-plugin": "^2.7.0",
+    "@typescript-eslint/parser": "^2.8.0",
+    "eslint": "^6.6.0",
+    "eslint-config-prettier": "^6.7.0",
```
Comment (andrewgordstewart) [inline packages/wallet-specs/package.json]: Should these be dev dependencies of the root repo? The root `eslintrc.js` uses them ...

File: packages/wallet-specs/src/ChannelStoreEntry.ts [hunk 2]
```diff
@@ -79,30 +73,28 @@ export class ChannelStoreEntry implements IChannelStoreEntry {
...
-      funding,
+      funding
     }: IChannelStoreEntry = this;
     return {
       supportedState,
       unsupportedStates,
       privateKey,
       participants,
       channel,
-      funding,
```
Comment (andrewgordstewart) [inline packages/wallet-specs/src/ChannelStoreEntry.ts]: I like my trailing commas. Not having them is really annoying in a few cases.

File: packages/wallet-specs/package.json (hunk 1 above)
Comment (snario) [reply packages/wallet-specs/package.json]: Probably, yes

File: packages/wallet-specs/src/ChannelStoreEntry.ts (hunk 2 above)
Comment (snario) [reply packages/wallet-specs/src/ChannelStoreEntry.ts]: I like trailing commas too. I'm in favour of turning them on in `packages/wallet-specs/.prettierrc.js` and eventually for all of the packages.

**Pull request merged**